src/components/weatherCard/weatherCard.js

```diff
@@ -1,13 +1,24 @@
-import React from "react";
-import { getPosition } from "../../helpers/getPosition";
-import { actionHandler } from "../../helpers/actionHandler";
+import React, { useState } from "react";
 import { useDispatch, useSelector } from "react-redux";
-import { Spinner, WeatherCardWrapper } from "./styles";
 import { apiKey } from "../../config";
+//
+import { getPosition } from "../../helpers/getPosition";
+import { actionHandler } from "../../helpers/actionHandler";
+import { WeatherCardWrapper } from "./styles";
+// Bootstrap
+import "bootstrap/dist/css/bootstrap.min.css";
+import { Card } from "react-bootstrap";
+import Spinner from "react-bootstrap/Spinner";
+import Collapse from "react-bootstrap/Collapse";
 
 export const WeatherCard = () => {
   const weatherDispatch = useDispatch();
   const weather = useSelector((state) => state);
```

Comment:
```suggestion
  const {isLoading, weatherData} = useSelector((state) => state);
```
check this, I'm not sure if this will be worked

Reply:
worked